tasks/release.sh

```diff
@@ -59,3 +59,4 @@ node ./node_modules/.bin/bundle-deps
 cd $root_path
 # Go!
 ./node_modules/.bin/lerna publish --independent "$@"
+./node_modules/.bin/lerna-npm-readme-hack --packages eslint-config-react-app
```

Comment:
oh, yeah. Somehow I've made an assumption that hack should be applied only to eslint-config-react-app, but didn't convey it.

Don't see why we shouldn't apply this to all packages